lib/Bitcode/Reader/BitcodeReader.cpp

```diff
@@ -4733,10 +4733,13 @@ const std::error_category &llvm::BitcodeErrorCategory() {
 
 static ErrorOr<std::unique_ptr<Module>>
 getBitcodeModuleImpl(std::unique_ptr<DataStreamer> Streamer, StringRef Name,
-                     BitcodeReader *R, LLVMContext &Context,
+                     std::unique_ptr<BitcodeReader> RPtr, LLVMContext &Context, // HLSL Change: unique_ptr
```

Comment:
Consider taking by `&&` to be consistent with other signatures.

Reply:
But inconsistent with the DataStreamer above? :)

Reply:
Ah, good point. Didn't even see that one :)